packages/react-components/react-nav-preview/src/components/NavCategory/NavCategory.test.tsx

```diff
@@ -5,5 +5,12 @@ describe('NavCategory', () => {
   isConformant({
     Component: NavCategory,
     displayName: 'NavCategory',
+    disabledTests: [
+      'component-handles-ref',
```

Comment:
NTH: comments about why Nav should disable these tests and if temporary or permanent.